cogs/StaffAppsUser.py

```diff
@@ -0,0 +1,567 @@
+                label="UTC+01:00",
+                description="Amsterdam, Berlin, Rome",
+            ),
+            discord.SelectOption(
+                label="UTC+02:00",
+                description="Cairo, Helsinki, Kaliningrad",
+            ),
+            discord.SelectOption(
+                label="UTC+03:00",
+                description="Moscow, Nairobi, Baghdad",
+            ),
+            discord.SelectOption(label="UTC+04:00", description="Abu Dhabi, Muscat"),
+            discord.SelectOption(
+                label="UTC+05:00",
+                description="Islamabad, Karachi, Tashkent",
+            ),
+            discord.SelectOption(label="UTC+06:00", description="Almaty, Dhaka"),
+            discord.SelectOption(
+                label="UTC+07:00",
+                description="Bangkok, Hanoi, Jakarta",
+            ),
+            discord.SelectOption(
+                label="UTC+08:00",
+                description="Beijing, Perth, Singapore",
+            ),
+            discord.SelectOption(label="UTC+09:00", description="Tokyo, Seoul, Osaka"),
+            discord.SelectOption(
+                label="UTC+10:00",
+                description="Brisbane, Canberra, Melbourne",
+            ),
+            discord.SelectOption(
+                label="UTC+11:00",
+                description="Magadan, Solomon Is., New Caledonia",
+            ),
+            discord.SelectOption(label="UTC+12:00", description="Auckland, Wellington"),
+            # Thank god for Copilot xoxo
+        ],
+    )
+    async def timezone_callback(
+        self,
+        select: discord.ui.Select,
+        interaction: discord.Interaction,
+    ) -> None:
+        self.timezone = select.values[0]
+        select.disabled = True
+        self.children[2].disabled = False
+        await interaction.response.edit_message(view=self)
+
+    @discord.ui.select(
+        placeholder="How many hours a week can you commit to the server?",
+        row=2,
+        min_values=1,
+        max_values=1,
+        disabled=True,
+        options=[
+            discord.SelectOption(
+                label="1 - 2 Hours",
+            ),
+            discord.SelectOption(
+                label="3 - 4 Hours",
+            ),
+            discord.SelectOption(
+                label="5 - 6 Hours",
+            ),
+            discord.SelectOption(
+                label="7 - 8 Hours",
+            ),
+            discord.SelectOption(
+                label="8 - 12 Hours",
+            ),
+            discord.SelectOption(
+                label="12 - 15 Hours",
+            ),
+            discord.SelectOption(
+                label="15 - 20 Hours",
+            ),
+            discord.SelectOption(
+                label="20+ Hours",
+            ),
+        ],
+    )
+    async def time_callback(
+        self,
+        select: discord.ui.Select,
+        interaction: discord.Interaction,
+    ) -> None:
+        self.time_per_wk = select.values[0]
```

Comment:
`__init__`?

Reply:
self.time_per_wk= select.values[0] is part of the callback when a user selects this option. It should not be included in the init.